bin/cml/send-comment.js

```diff
@@ -27,6 +25,25 @@ exports.builder = (yargs) =>
         default: 'HEAD',
         description: 'Commit SHA linked to this comment'
       },
+      publish: {
+        type: 'boolean',
+        description:
+          'Upload local files and images linked from the Markdown report'
+      },
+      watch: {
+        type: 'boolean',
+        description: 'Watch for changes and automatically update the report'
+      },
+      triggerFile: {
+        type: 'string',
+        description: 'File used to trigger the watcher',
+        hidden: true
+      },
+      native: {
+        type: 'boolean',
+        description:
+          "Uses driver's native capabilities to upload assets instead of CML's storage. Not available on GitHub."
```

Comment:
```suggestion
          "Uses driver's native capabilities to upload assets instead of CML's storage. Not available on GitHub"
```

Reply:
Moved to https://github.com/iterative/cml/issues/762#issue-1024172986